src/Sds011.cpp

```diff
@@ -138,9 +138,16 @@ bool Sds011::crc_ok() {
     for (int i = 2; i < 8; ++i) {
         crc += _buf[i];
     }
+    if (crc != _buf[8]) checksum_errors++;
```

Comment:
I prefer pre-increment unless the effect of post-increment is explicitly required.